### PR TITLE
Experimental update

### PR DESCRIPTION
**Pull request by evevkovacs**
This PR makes minor changes in the code to run the production pipeline

### CODE REVIEW

File: diffsky/experimental/photometry_interpolation.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 """
 """
-from dsps.sed import calc_ssp_weights_sfh_table_lognormal_mdf
```
Comment (aphearin) [inline diffsky/experimental/photometry_interpolation.py:3]: This import statement works as written with dsps v0.3

File: diffsky/experimental/photometry_interpolation.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+from dsps.sed.ssp_weights import calc_ssp_weights_sfh_table_lognormal_mdf
 from jax import vmap, jit as jjit
-from dsps.cosmology import age_at_z
```
Comment (aphearin) [inline diffsky/experimental/photometry_interpolation.py:5]: This import statement works as written with dsps v0.3

**Review by aphearin — COMMENTED** [review]
@evevkovacs could you explain why these changes are necessary? Both of these import statements for in the v0.3 release of DSPS

File: diffsky/experimental/photometry_interpolation.py (hunk 2 above)
Comment (evevkovacs) [reply diffsky/experimental/photometry_interpolation.py:5]: In the version of diffsky that I had, neither of these imports worked. The code was:
from dsps import age_at_z
from dsps import calc_ssp_weights_sfh_table_lognormal_mdf
I had to make the same modifications that are now in the latest version of diffsky, so these changes are redundant. Sorry, I didn't realize that the update had already happened.

**Pull request closed without merging**